src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx

```diff
@@ -80,13 +86,14 @@ The default view provides insight into three key service performance indicators
 
 ### Circle visualization and table view [#circles]
 
-Each application (or other facet) is represented by a circle. The size of a circle indicates the magnitude of the deviation of the signal for that application. The circle color indicates whether the value has significantly decreased or increased in the last five minutes, based on the standard deviation of the prior hour (default evaluation and comparison time windows.) Entities are sized, colored, and ordered by rate of deviation. A maximum of 500 entities will be displayed at a time. You can filter your data to focus your analysis.
+Each application (or other facet) is represented by a circle. The size of a circle indicates the magnitude of the deviation of the signal for that application. The color of the circle indicates whether the value has significantly decreased or increased in the last five minutes, based on the standard deviation of the prior hour (default evaluation and comparison time windows.) 
```

Comment:
I know deviation is a stat term, but I'm not sure if we need to define it a little better. Is the deviation referring to the difference in performance between the last 5 minutes and the previous hour? 

Only bringing it up because it appears throughout the draft